tony-core/src/main/java/com/linkedin/tony/Utils.java

```diff
@@ -404,4 +403,8 @@ public static void addResource(String path,
       LOG.error("Failed to add " + path + " to local resources.", exception);
     }
   }
+
+  public static String generateFileName(TonyApplicationMaster.Metadata obj) {
```

Comment:
obj is not a good name..

Reply:
See https://github.com/linkedin/TonY/pull/79#discussion_r233593949.